prestodb/client.py

```diff
@@ -301,14 +304,34 @@ def next_uri(self):
         # type: () -> Text
         return self._next_uri
 
-    def post(self, sql):
-        return self._post(
+    def post(self, sql, allow_redirect=False):
```

Comment:
maybe `self._redirect_handler is not None` is good enough; users do not have ability to make this `True`